package.json

```diff
@@ -7,6 +7,7 @@
     "npm": ">=7.0.0"
   },
   "dependencies": {
+    "@ltd/j-toml": "^1.38.0",
```

Comment:
I've just added this one in addition to the other package that we have. The existing `toml` package only parses TOML, it isn't bi-directional.

I could replace the TOML package completely, but maybe that would be better for a follow up PR